frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -129,9 +133,18 @@ type cvdInstance struct {
 	Displays       []string `json:"displays"`
 	InstanceDir    string   `json:"instance_dir"`
 	WebRTCDeviceID string   `json:"webrtc_device_id"`
+	ADBSerial	   string   `json:"adb_serial"`
```

Comment:
nit: format the code with `go fmt`

Reply:
Done in https://github.com/google/android-cuttlefish/pull/522.